### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by misteraverin**

### CODE REVIEW

File: api_controller/views.py [hunk 1]
```diff
@@ -4,37 +4,16 @@
...
-        ret["results"] = data
-        return Response(ret)
+from api_controller.serializers import PaginationDecorator
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from rest_framework import serializers
 
 
 class BasicApiViewSet(viewsets.GenericViewSet):
     serializer_class = None
+    paginate_by = 3  # how many objs for displaying on 1 page
```
Comment (KirillMakarov) [inline api_controller/views.py]: Why only 3, can we show 20?

File: translation/models/translation_services.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def create_text_translation(self, text, **kwargs):
         # if text is long html, &lang param isn't parsed properly
         params.append("&{0}={1}".format("text", text))
         response = requests.get(final_url + "".join(params)).json()
-        return response['text']
+        return " ".join(response['text'])
```
Comment (KirillMakarov) [inline translation/models/translation_services.py:36]: here is symbol ` ` before `response['text']`. Is it matter to use `""` or `" "`?

**Review by KirillMakarov — CHANGES_REQUESTED** [review]

File: api_controller/views.py (hunk 1 above)
Comment (misteraverin) [reply api_controller/views.py]: ok

File: translation/models/translation_services.py (hunk 2 above)
Comment (misteraverin) [reply translation/models/translation_services.py:36]: Actually, `response['text']` is a list with single element, but for sure I use " " if we have 2 or more elements and don't want to mess them.

Comment (KirillMakarov) [reply translation/models/translation_services.py:36]: ok

**Pull request merged**